### PR TITLE
Twitter-link

### PR DESCRIPTION
**Pull request by HeniMKH**
## Naming Rules

Name your PR like this: ISSUENUMBER-TITLE-YOURNAME

## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

**Review by karleenmsrichards — COMMENTED** [review]
This pull request has been reviewed and is good.

**Review by farzaneh-haghani — APPROVED** [review]
Approved.

File: client/src/pages/Home.jsx [hunk 1]
```diff
@@ -38,7 +39,17 @@ export function Home() {
...
+	href="https://www.Twitter.com"
+	target="_blank"
+	rel="noopener noreferrer"
+	>
+	<TwitterIcon style={{ fontSize: 40, color: "blue" }} />
+	</IconButton>
+	
 	</Box>
 			</div>
 		<Footer />
```
Comment (jarrodruby) [inline client/src/pages/Home.jsx:55]: the formatting doesn't look right. Can we use eslint https://chariotsolutions.com/blog/post/configure-visual-studio-code-lint-react-jsx-syntax/

**Review by jarrodruby — CHANGES_REQUESTED** [review]

File: client/src/pages/Home.jsx (hunk 1 above)
Comment (HeniMKH) [reply client/src/pages/Home.jsx]: Done

**Review by karleenmsrichards — APPROVED** [review]
Changes have been reviewed and approved.

**Pull request closed without merging**